pkg/cidata/cidata.TEMPLATE.d/boot/40-install-containerd.sh

```diff
@@ -28,7 +28,7 @@ if [ ! -f "${LIMA_CIDATA_GUEST_INSTALL_PREFIX}"/bin/nerdctl ] || [[ "${tmp_extra
 			sudo -iu "${LIMA_CIDATA_USER}" "XDG_RUNTIME_DIR=/run/user/${LIMA_CIDATA_UID}" "PATH=${PATH}" containerd-rootless-setuptool.sh uninstall
 		)
 	fi
-	tar Cxzf "${LIMA_CIDATA_GUEST_INSTALL_PREFIX}" "${LIMA_CIDATA_MNT}"/nerdctl-full.tgz
+	tar Cxaf "${LIMA_CIDATA_GUEST_INSTALL_PREFIX}" "${LIMA_CIDATA_MNT}"/"${LIMA_CIDATA_CONTAINERD_ARCHIVE}"
```

Comment:
How is "a" old?
Does this work with EL8, Ubuntu 20.04, etc.?

Reply:
Found: It is from 2007
https://git.savannah.gnu.org/cgit/tar.git/commit/?h=release_1_20&id=620a136e74f211c2fb98b45fe44261d5d775f6a9